### DISCUSSION TITLE
In about 24 hours only at 31% plot

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#4606] [Support]
I've tried multiple variations of settings, etc and I still get estimated time frames of 2-4 days to create a standard 101gb plot.  5th gen i7 with 32gigs of ram and trying to plot on 4 ssds in Raid 0.  Also tried on standard 500gb sata drive and a 6tb sata drive. all seem to work at the same speed, never use up all the ram allocated to them (as confirmed by task manager threads), nor utilize much of the processor.  At about 24 hrs, my plots are usually around 20-30% which I've usually been cancelling due to it being, clearly, wrong compared to other people's estimates.  Currently, at 31% after about 24 hrs.

**Comment by heeroz** [comment]
You should check the plot log at first.

**Reply by number435398** [reply]
Check it for what?

**Reply by zjshen14** [reply]
The progress percentage is not linear. It will stay at 31% for pretty long.

One problem I could think of from your description is that you're using sata instead of nvme. Long in short nvme is way faster than sata. You could google the difference.

**Reply by number435398** [reply]
Not being linear is good to know, but also makes the percentages only about half as valuable.

I am using SATA, but my Raid 0, in testing, is just about as fast as nvme. My nvme is coming in tomorrow.

Either way, generally estimates I've seen online max out at about 12 hours to plot a 101gb plot. I've more than doubled that time with plenty of ram and processing power leftover that isn't being used.

**Reply by number435398** [reply]
Why don't I ask this; what kind of speeds are other people actually getting for plotting a 101gb plot?  Maybe my goals are unrealistic

**Comment by dslrnation** [comment]
I had the same problem and I fixed it by plotting without parallel, I test 3 plots at a time and each one has 2 in queue (2 threads, 3800 mib, each plot named differently so they can start at the same time ( if you have the same name they will all be in queue) total time for 3 plots to finish about 9 hours, I have 12 threads and 32gb ram Laptop￼￼

**Reply by number435398** [reply]
I'm confused by "3 plots at a time", you do no more than 3 plots in parallel, or are plotting 1 and then putting 2 in the queue?  Are you using an NVME drive for the temp directory? What operating system are you using?  I'm using Windows 7 Ultimate, 64bit

**Reply by dslrnation** [reply]
I do the regular plot not the parallel option, so I select 3 and create the plot then it starts and puts 2 in queue, and then I go and do the same two more times so I can get 3 starting at the same time, windows 10, I uses 1tb Sandisk extreme via type c which is not a great option but it worked, no I got 2tb nvme to make 5 plots at the same time, testing it now

**Reply by number435398** [reply]
Okay.  I've just started using an NVME, and its not much better than my sata plate drive. I'm doing a bunch in parallel, but some people measure their plotting in "TB per day", and I'm not even getting 1tb per day, before, or after, my nvme.

**Reply by dslrnation** [reply]
Parallel didn't work for me, I measured phase one and I added 40 minutes on top of 240 minutes for delay and still it freezes at 30%, each nvme has a different speed, mine is 5000MBps if hooked internally, but I'm using it with a usb enclosure so it gives me about 680MBps

**Reply by number435398** [reply]
my nvme is rated at 3gb/s, my motherboard rated it at 4x, pci-e 2.0.  It seems strange to me that some people are claiming, at least 1 tb per day (if not more) and I might get 0.5tb with parallel. One of the points someone made is that windows has a 5-10% increase in plotting times than other OSs, but I'm clearly well beyond that.

**Comment by number435398** [comment]
Update: I just installed an nvme drive and began plotting.  Plotting on my nvme vs 4 sata SSDs in raid 0 (just the small 101gb plots) seems to be marginally better.  Still estimating around 24hrs to complete a plot.  Chia processes have yet to utilize all available RAM and all available CPU usage, though they keep claiming they are in the logs.  Logs claim things such as "CPU (109.340%)".  

I've got over 20TB of space available to use for Chia Coin, but I can't seem to plot it fast enough. Its like my current available space is going to become obsolete on the network by the time I plot all that space while others are breezing through this stuff.

**Reply by dslrnation** [reply]
My best advice is to watch and read lots of videos and posts, that's what I did, I failed many times and know I know enough to plot but there's lot more to learn, for example I'm not sure if I'm farming correctly now because my challenges are empty in the farm section, so more and more research

**Reply by number435398** [reply]
Thank you for your advice.  Its good to note, but it also stresses another issue: is this a coin where you have to do tons and tons of research to be efficient, or can just any n00b just jump in? Or rather, not is it such a coin, but is it _**intended**_ to be such a coin?

**Reply by dslrnation** [reply]
you're welcome, not really but For me  for example I want to know as much as I can about the errors and warnings so I can read the logs and understand what should I fix, BTW I Wa able to create 5 plots at the same time it took about 9 to 10 hours for all to finish

**Comment by number435398** [comment]
Update: I, very begrudgingly, upgraded to Windows 10 and my drive speed doubled from about 600mb/s to around 1200mb/s, at least according to Novabench.  CrystalDiskMark says I'm still at about 600mb/s. So, I'm guessing that's my bottleneck and not the Chia program. I would like to note that adding a write-speed number in the program would help narrow such concerns for others in the future. Also turns out my motherboard doesn't like my nvme drive to work at pci-e 3.0 and limits it to 2.0, even though I had to give up a whole PCI-E slot to even use it

**Reply by dslrnation** [reply]
There's a guy on YouTube he built a $650 with Xeon cpu and this price without the firecuda nvme and he said it will create 35 plots per day and also the motherboard needs a pcie adapter for the nvme, I actually did when he recommended and I'll receive the pc today, I'll test it and check the results in a day or 2, and yes upgrading to windows 10 necessary￼

**Reply by number435398** [reply]
Can you provide any links to exactly what you're referencing please?  Tons of videos on YouTube to filter through to find that particular example.

**Reply by number435398** [reply]
I found several videos that seem to be what you were referencing, but I don't know what's so special about that setup. I got a WD_Black PCIe card, and though it benchmarks much higher than my nvme ssd that was plugged into a 2.0 x4 slot (while the black was in a 3.0 x16 slot) the speed of the plotting was slightly slower (by between about 15 minutes to an hour, I hadn't measured the previous ones accurately enough). For some reason the real-time transfer rates to my WD_Black while plotting don't exceed approx 250MB/s even though it benchmarks at nearly 4GB/s

**Reply by pluto2021** [reply]
I would guess that is a common problem. The benchmarks are for sequential R/W. I believe that the plotting is kind of random accessing to the NVMe.  When using CrystalDiskMark, check the random access read/write speeds, it is about one order of magnitude slower than the sequential speed!

**Reply by number435398** [reply]
But if that's the case, @pluto2021 then what's the point of getting some incredibly fast NVMe SSD or the like?  Its almost as if we're all convinced that it matters when, if the speeds are really more like random access read/write speeds, we're all just wasting our money

**Reply by pluto2021** [reply]
It seems that the random access speed is a technology hurdle for the SSD industry (actually worse for HDDs). If you check the SSD R/W speed in the task manager, you will see that most of the time, it is fluctuating randomly, no where near any sequential mode. Maybe this SSD/HDD mania can help to fund the next gen SSD technology that can fix the random access speed bottleneck in general, that can help other applications other than the Chia blockchain as well.  I hope that the SSD industry can make the SSDs as cheap as the HDDs sooner with all these new revenues they could never dream of if it were not because of the Chia blockchain development.

Sadly people get bought in by the genius marketing of the SSD industry, only showing the best part, yet hiding the ugly truth behind the scene. Imagine that they publish all those real specs that would matter most of the time, would you still have that kind of urge to buy or be willing to pay the extra premium for the top sequential speed? At least you may think twice. Many brands of SSDs would put a fast block of SLC before the really slow bulk part of the SSD capacity. When you test under any normal programs like CrystalDiskMark, BlackMagic etc, you will be very happy, yet, it is hard to find a program accessible to consumers to test those slooowww portion of the SSD space!  It is all marketing. Some really cheap (not really, only relatively) brands would put really slow SSD space hiding behind the fast SLC. To lower the cost of production yet still able to show the highest speed, you can put the slowest (the cheapest) SSD behind the SLC curtain without being detected by CrystalDiskMark or BlackMagic and the like. So in general, you pay for what you get, yet even the highest cost SSDs cannot beat the random access bottleneck.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
